languages/elisp/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"github.com/replit/prybar/utils"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+func Execute(config *utils.Config) {
+
+	emacs, err1 := exec.LookPath("emacs")
+
+	if err1 != nil {
+		panic(err1)
+	}
+
+	execPath, err2 := os.Executable()
```

Comment:
just keep modifying the same err, `:=` will set any existing variables

Reply:
Fixed, thanks.